writeit/templates/subdomains/iran/write/missing_contacts.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base_instance.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block promo %}
+{% endblock %}
+
+
+{% block content_inner %}
+
+    <div class="full-width-wrapper write-wrapper">
+      <main class="centred write">
+
+        <h1>{% trans "Help to find missing contact details" context "Title of page appealing for help finding more contact details" %}</h1>
+
+        <p>
+          {% blocktrans trimmed with email='support@namehbeanha.com' %}
+            We don't have an email address for any of the people
```

Comment:
Same comment about curly quotes, as in `nuntium/templates/write/missing_contacts.html`